packages/runtime-dom/src/apiCustomElement.ts

```diff
@@ -313,10 +330,12 @@ export class VueElement extends BaseClass {
   }
 
   protected _setAttr(key: string) {
-    let value = this.getAttribute(key)
+    let value = this.getAttribute(key) as any
```

Comment:
Maybe we can determine that the type of value is boolean, string and number
https://github.com/vuejs/core/pull/9698/files#diff-a76582b8bbc7775f5a45fe4518abb808080429939a0102d3e768dfa17be6f172R366

Reply:
Updated type, thank you.